.github/workflows/prepare-package-release.yml

```
@@ -25,3 +25,68 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: echo "Preparing release for ${{inputs.package}}"
+      - uses: kanga333/variable-mapper@master
+        with:
+          key: "${{inputs.package}}"
+          map: |
+            {
+              "@realm/babel-plugin": {
+                "package-path": "packages/babel-plugin",
+                "package-name": "@realm/babel-plugin",
+                "version-prefix": "babel-plugin-"
+              },
+              "@realm/common": {
+                "package-path": "packages/common",
+                "package-name": "@realm/common",
+                "version-prefix": "realm-common-"
+              },
+              "@realm/network-transport": {
+                "package-path": "packages/network-transport",
+                "package-name": "@realm/network-transport",
+                "version-prefix": "realm-network-transport-"
+              },
+              "@realm/react": {
+                "package-path": "packages/react",
+                "package-name": "@realm/react",
+                "version-prefix": "realm-react-"
+              },
+              "@realm/tools": {
+                "package-path": "packages/tools",
+                "package-name": "@realm/tools",
+                "version-prefix": "realm-tools-"
+              },
+              "@realm/web": {
+                "package-path": "packages/web",
+                "package-name": "realm-web",
+                "version-prefix": "realm-web-"
+              }
+            }
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      # action is at https://github.com/realm/ci-actions/tree/main/update-changelog
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@main
+        with:
+          version: ${{ inputs.version }}
+          changelog: ${{ github.workspace }}/${{env.package-path}}/CHANGELOG.md
+
+      - name: Update package.json
+        working-directory: ${{ github.workspace }}/${{env.package-path}}
+        run: |
+          jq '.version="${{ steps.update-changelog.outputs.new-version }}"' package.json > package.tmp
+          mv package.tmp package.json
+
+      - name: Install Dependencies and update package-lock.json
+        run: |
+          npx lerna bootstrap --scope=${{ env.package-name }} --include-dependencies
+
+      - name: Create Release PR
+        uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #! 3.10.1
```

We might as well use the latest version?

```suggestion
        uses: peter-evans/create-pull-request@v4
```